NineChronicles.Headless/GraphTypes/AddressQuery.cs

```diff
@@ -68,7 +75,38 @@ public AddressQuery()
                         Description = "world boss season id."
                     }
                 ),
-                resolve: context => Addresses.GetRaiderListAddress(context.GetArgument<int>("raidId")));
+                resolve: context => Addresses.GetRaiderListAddress(
+                    context.GetArgument<int>("raidId")));
+
+            Field<ListGraphType<NonNullGraphType<AddressType>>>(
+                name: "currencyMintersAddress",
+                description: "currency minters address.",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<CurrencyEnumType>>
+                    {
+                        Name = "currency",
+                        Description = "A currency type. " +
+                                      "see also: https://github.com/planetarium/NineChronicles.Headless/blob/main/NineChronicles.Headless/GraphTypes/CurrencyEnumType.cs",
+                    }),
+                resolve: context =>
+                {
+                    var currency = context.GetArgument<CurrencyEnum>("currency");
+                    switch (currency)
+                    {
+                        case CurrencyEnum.NCG:
+                            var blockchain = standaloneContext.BlockChain!;
+                            var goldCurrencyStateDict =
+                                (Dictionary)blockchain.GetState(Addresses.GoldCurrency);
+                            var currencyDict = (Dictionary)goldCurrencyStateDict["currency"];
+                            var mintersList = (List)currencyDict["minters"];
+                            return mintersList.Select(minterValue =>
```

Comment:
How about using `GoldCurrencyState`? Is there a reason to get currency manually?